composer.json

```diff
@@ -71,13 +69,21 @@
       "locations": {
         "web-root": "drupal/"
       }
+    },
+    "enable-patching": true,
+    "composer-exit-on-patch-failure": true,
+    "patches": {
+        "drupal/drupal-driver": {
+            "https://github.com/jhedstrom/DrupalDriver/pull/253": "https://patch-diff.githubusercontent.com/raw/jhedstrom/DrupalDriver/pull/253.diff"
```

Comment:
Instead of patching, I'll merge https://github.com/jhedstrom/DrupalDriver/pull/253 and then we can update composer.json to pull dev until we release 5.0.